backend/src/appointment/controller/mailer.py

```diff
@@ -251,8 +251,8 @@ def html(self):
             duration=self.duration,
             # Image cids
             tbpro_logo_cid=self._attachments()[0].filename,
-            calendar_icon_cid=self._attachments()[1].filename,
-            clock_icon_cid=self._attachments()[2].filename,
+            calendar_icon_cid=self._attachments()[2].filename,
+            clock_icon_cid=self._attachments()[3].filename,
```

Comment:
We should maybe switch to a dict, or change the BaseBookingMail so that inline images are separate from other attachments.

Reply:
I'm fine with both! Can you put that into a follow-up ticket? Thanks!

Reply:
https://github.com/thunderbird/appointment/issues/860 🎉

Reply:
Thank you!